test/unit/specs/components/staking/__snapshots__/ModalWithdrawAllRewards.spec.js.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ModalWithdrawAllRewards has the expected html structure 1`] = `
+<action-modal-stub
+  class="modal-withdraw-rewards"
+  id="modal-withdraw-all-rewards"
+  submissionerrorprefix="Withdrawal failed"
+  submitfn="function () { [native code] }"
+  title="Withdraw"
+  validate="function () {
+        /* istanbul ignore next */
+        cov_6em7wsq1m.f[8]++;
```

Comment:
@faboweb this line fails on CI, any idea why ?

Reply:
Istanbul is messing with the snapshots if we use inline functions in the Dom. I have no immediate solution